styles/style.css

```diff
@@ -0,0 +1,448 @@
+  color: #fff;
+  border-bottom: 6px solid rgba(255, 255, 255, 0.349);
+}
+
+h2 {
+  font-family: "'Open Sans'",sans-serif;
+  font-weight: 300;
+  color: #a7a7a7;
+  text-align: center;
+}
+
+header img {
+  float: left;
+  width: 180px;
+  margin: 14px;
+}
+
+p {
+  max-width: 960px;
+  margin: 20px auto;
+  color: #fff;
+}
+
+p a {
+  text-decoration: none;
+  color: #0074D9;
+  transition: color 0.4s;
+}
+
+p a:hover {
+  color: #7FDBFF;
+  cursor: pointer;
+}
+
+p a:active {
+  color: #007BE6;
+}
+
+.intro .intro-info a {
+  color: #fdf3d6;
+}
+
+.intro .intro-info a:hover {
+  color: #ddd;
+  cursor: pointer;
+}
+
+.intro .intro-info a:active {
+  color: #fdf3d6;
+}
+
+form {
+  border-radius: 5px;
+  padding: 10px;
+  max-width: 960px;
+  margin: 40px auto;
+  box-shadow: rgba(0, 0, 0, 0.2) 0px 4px 8px 0px;
+  background: #fff;
+  position: relative;
+}
+
+form label {
+  width: 90%;
+  padding: 12px 0px 0px;
+  margin: 16px auto 0;
+  display: block;
+  font-size: 15px;
+}
+
+form label[for=download] {
+  margin-bottom: 6px;
+}
+
+form input[type=text],
+form input[type=number],
+form input[type=email],
+form input[type=date],
+form input[type=time],
+form input[type=file],
+form input[type=image],
+form textarea {
+  width: 90%;
+  padding: 12px 0px;
+  margin: 8px auto;
+  display: block;
+  border: none;
+  outline: 0;
+  border-bottom: 2px solid #ccc;
+  box-sizing: border-box;
+  transition: border-bottom 0.1s;
+  font-size: 15px;
+}
+
+form input[type=text].missing,
+form input[type=number].missing,
+form input[type=email].missing,
+form input[type=time].missing,
+form input[type=file].missing,
+form textarea.missing {
+  border-bottom: 2px solid #e00122;
+}
+
+form h3 {
+  width: 90%;
+  padding: 12px 0px 0px;
+  margin: 16px auto 0;
+  display: block;
+  font-size: 15px;
+}
+
+form p {
+  color: #000;
+  width: 90%;
+  padding: 12px 0px 0px;
+  margin: 0 auto 4px;
+  font-size: 14px;
+}
+
+form input[type=text]:focus,
+form input[type=number]:focus,
+form input[type=email]:focus,
+form input[type=time]:focus,
+form input[type=file]:focus {
+  border-bottom: 2px solid #8f8f8f;
+}
+
+form input[type=text].missing:focus,
+form input[type=number].missing:focus,
+form input[type=email].missing:focus,
+form input[type=time].missing:focus,
+form input[type=file].missing:focus,
+form textarea.missing {
+  border-bottom: 2px solid #8f131f;
+}
+
+form input[type=button] {
+  background-color: #e00122;
+  font-size: 20px;
+  text-transform: uppercase;
+  transition: background-color 0.6s, color 0.6s;
+  margin: 20px auto;
+  display: block;
+  width: 90%;
+  padding: 10px;
+  -webkit-appearance: none;
+  outline: 0;
+  border: 2px solid #e00122;
+  border-radius: 4px;
+  color: #fff;
+}
+
+form input[type=button]:hover {
+  background-color: #fff;
+  color: #e00122;
+  border: 2px solid #e00122;
+  cursor: pointer;
+}
+
+form input[type=file] {
+  visibility: hidden;
+  position: absolute;
+}
+
+form .file-upload {
```

Comment:
Can you style this according to the [updated Luau Tickets](https://github.com/mihi-r/ceas-tribunal-luau-tickets/blob/master/styles/style.css#L250) (or Resume Review Day)? It will be really nice to get everything to match.

Reply:
Styling matches what's on the [live site](https://tribunal.uc.edu/luau-tickets/).